app/controllers/api/v1/mattermost_controller.rb

```diff
@@ -1,4 +1,7 @@
 class Api::V1::MattermostController < Api::ApplicationController
+  require 'commands'
```

Comment:
It's recommended to use [`require_dependency`](https://apidock.com/rails/ActiveSupport/Dependencies/Loadable/require_dependency) when requiring another part of the application from an autoloaded file like this (it helps with code reloading). Otherwise this looks good!

Reply:
Ah good to know!  I didn't understand that `require` doesn't pick up changes to the required file.